doc/concepts/decor.rst

```diff
@@ -287,6 +287,37 @@ See :ref:`modelgapjunctions`.
 4. Stimuli
 ~~~~~~~~~~
 
+A current stimilus is a DC or sinusoidal current of fixed frequemcy with a time-varying amplitude
+governed by a piecewise-linear envelope.
+
+The stimilus is described by two parameters: a frequency in Hertz, where a value of zero denotes DC;
```

Comment:
Typo: stimulus

Reply:
I am particularly impressed by how I managed to mistype it twice.